javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,73 @@
+//Age-ify (A future age calculator)
+let yearOfBirth = 1997;
+let yearFuture = 2055;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+let dogYearOfBirth = 2019;
+let dogYearFuture = 2030;
+let dogYear = 0.25;
+/* the calculation of dog and human years are not clear for me,
+ so I assumed that 1 dog year equals 4 human years*/
+let shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears === true) {
+  let dogAge = (dogYearFuture - dogYearOfBirth) * dogYear;
+  console.log(
+    "Your dog will be " + dogAge + " dog years old in " + dogYearFuture
+  );
+} else {
+  let dogAge = dogYearFuture - dogYearOfBirth;
+  console.log(
+    "Your dog will be " + dogAge + " human years old in " + dogYearFuture
+  );
+}
```

Comment:
According to Google, it is widely assumed that one dog year is estimated to 7 human years. However, specialists are arguing that the first and second year of a dogs life can be calculated to respectively 15 and 9 human years, depending on the dog size and race etc. It is apparently not so simply to calculate. 😇

Aside from that, the logic of your code is spot on. It’s great to see you using comments to make your code clear and readable. 

/** Observations & Suggestions **/
It’s good to see you declare and assign your variables at the top of your dog age calculator. 👌 I would argue it could become even more clean if you place your boolean “shouldShowResultInDogYears” before the multi-line comment. Then you have your variable assignments nicely structured, side by side.  

Could some of your variables in this assignment be constants?